src/components/OptionalLink.tsx

```diff
@@ -0,0 +1,30 @@
+import type { CSSObject } from '@mui/material';
+import NextLink from 'next/link';
+import type { PropsWithChildren } from 'react';
+
+interface OptionalLinkProps {
+  url?: string;
+  alt?: string;
+  styles?: CSSObject;
+}
+
+export const OptionalLink: React.FC<PropsWithChildren<OptionalLinkProps>> = (
+  props,
+) => {
+  if (props.url) {
+    return (
+      <NextLink
```

Comment:
can you do it as mui component so we keep theming? e.g. https://github.com/jumperexchange/jumper-exchange/pull/1344/files#diff-927fb0b729f8d1aaff94ec40b4bfe2d47f24e69dd48e9fb1fb2ba502028ba91eR145

Also, url -> href (keep same name for consistency)
styles -> sx

etc..

Reply:
Good idea! Done